ts/main.ts

```diff
@@ -33,11 +45,87 @@ function handleSubmit(event: SubmitEvent): void {
     photoURL: $photoURL.value,
     notes: $entryNotes.value,
   };
-
   data.nextEntryId += 1;
   data.entries.unshift(newFormEntry);
 
+  toggleNoEntries();
+  // window.location.reload();
   $img?.setAttribute('src', 'images/placeholder-image-square.jpg');
   $form?.reset();
   writeData();
+  renderEntry(newFormEntry);
+  viewSwap('entries');
+}
+const storedData = readData();
+Object.assign(data, storedData);
```

Comment:
You shouldn't be doing this here. You call `readData` and assign it to `data` in `data.ts`